python/deltalake/table.py

```diff
@@ -45,18 +63,38 @@ def files_by_partitions(self, partition_filters: List[Tuple]) -> List[str]:
             )
 
     def file_paths(self) -> List[str]:
+        """
+        Get the list of files with an absolute path.
+        :return: list of the .parquet files with an absolute path referenced for the current version of the DeltaTable
+        """
         return self._table.file_paths()
 
     def load_version(self, version: int) -> None:
+        """
+        Load a DeltaTable with a specified version.
+        :param version: the identifier of the version of the DeltaTable to load
+        """
         self._table.load_version(version)
 
     def schema(self) -> Schema:
+        """
+        Get the current schema of the DeltaTable
+        :return: the current Schema registered in the transaction log
+        """
         return Schema.from_json(self._table.schema_json())
 
     def pyarrow_schema(self) -> pyarrow.Schema:
+        """
+        Get the current schema of the DeltaTable with the Parquet PyArrow format.
+        :return: the current Schema with the Parquet PyArrow format
+        """
         return pyarrow_schema_from_json(self._table.arrow_schema_json())
 
     def to_pyarrow_dataset(self) -> pyarrow.dataset.Dataset:
+        """
+        Read the DeltaTable with the PyArrow dataset format.
```

Comment:
Using build to suggest that the resulting Dataset is self-contained and has no reference to the original delta table anymore.

```suggestion
        Build a PyArrow Dataset using data from the DeltaTable.
```